modules/gdscript/gdscript_analyzer.h

```diff
@@ -107,7 +107,7 @@ class GDScriptAnalyzer {
 
 	// Helpers.
 	GDScriptParser::DataType type_from_variant(const Variant &p_value, const GDScriptParser::Node *p_source);
-	GDScriptParser::DataType type_from_metatype(const GDScriptParser::DataType &p_meta_type) const;
+	static GDScriptParser::DataType type_from_metatype(const GDScriptParser::DataType &p_meta_type);
```

Comment:
Changing this to `static` is not really necessary (though not harmful either). It would a better improvement to actually remove it from the class and keep it as static standalone function in the `cpp` file.

Reply:
Yes, I think at some point a little refactor making sure anything not using Parser or Analyzer state is made class static or file static depending on whether it's used outside a single file makes a lot of sense! Thanks for looking into the PR :)